catalog/app/utils/s3paths.js

```diff
@@ -179,7 +179,7 @@ export const handleFromUrl = (url, referrer) => {
 export const handleToHttpsUri = ({ bucket, key, version }) =>
   `https://${bucket}.s3.amazonaws.com/${encode(key)}${mkSearch({ versionId: version })}`
 
-export const handleToS3Url = ({ bucket, key, version }) =>
+export const handleToS3Url = ({ bucket, key, version = undefined }) =>
```

Comment:
Isn't it `undefined` by default?

Reply:
it is, but TS complains, it thinks if you have a var in destructuring pattern, it should be defined / required, unless signified explicitly that it has a default value. converting this file to TS would solve the problem.